packages/flame/lib/src/text/formatters/text_painter_text_formatter.dart

```diff
@@ -1,22 +1,22 @@
+import 'package:flame/src/text/elements/debug_text_painter_text_element.dart';
+import 'package:flame/src/text/elements/text_painter_text_element.dart';
 import 'package:flame/src/text/formatters/text_formatter.dart';
-import 'package:flame/src/text/inline/debug_text_painter_text_element.dart';
-import 'package:flame/src/text/inline/text_painter_text_element.dart';
-import 'package:flutter/rendering.dart';
+import 'package:flutter/rendering.dart' as flutter;
```

Comment:
How come this is needed everywhere now? If there is a clash between two classes I would `hide` the one that isn't used instead.

Reply:
Ok, i removed the prefix.